### ISSUE TITLE
[Suggestion] Add a static / radio effect option

### ISSUE THREAD

**Issue by TheVeggum** [Flaton1/walkie-talkie-mod#35]
**Is your feature request related to a problem? Please describe.**
The walkies sound way too clear in my opinion and dont really give off the impression you're talking on a radio.

**Describe the solution you'd like**
It would be really cool to have an option you can enable that makes the walkies actually sound like a radios in game. I think that having a bit of static / distortion to the walkies, especially at longer ranges if possible, would make the mod a lot more immersive instead of just sounding like you're on a discord call together.

**Describe alternatives you've considered**
Alternatives to this would be me and my friends having to use voice changers out of the game and make those our input devices but that is effort a lot of people aren't willing to go through.

**Additional context**
N/A

**Comment by MoutonBinoclard** [comment]
I agree, also it would be easier to know if the sound comes from proxy chat, or from the talkie.
Would be very useful

**Comment by 32bitx64bit** [comment]
Simple radio does this, at least I believe so.
Its a much more complex mod, has AM / FM, lots of features, and walkie talkies (Transceiver)
